pyudmx/pyudmx.py

```diff
@@ -75,7 +77,8 @@ def close(self):
             usb.util.dispose_resources(self._dev)
             self._dev = None
 
-    def _send_control_message(self, cmd, value_or_length=1, channel=1, data_or_length=1):
+    def _send_control_message(self, cmd: Union[1, 2], value_or_length: int = 1, channel: int = 1,
+                              data_or_length: Union[1, bytearray] = 1) -> int:
```

Comment:
Same issue with Union here. This should be data_or_length: Union[int, bytearray] = 1